k8s/influxdb/values.yaml

```diff
@@ -0,0 +1,256 @@
+  #    requests:
+  #      memory: 512Mi
+  #      cpu: 2
+  #    limits:
+  #      memory: 1Gi
+  #      cpu: 4
+
+## Create default user through Kubernetes job
+## Defaults indicated below
+##
+setDefaultUser:
+  enabled: false
+
+  ## Image of the container used for job
+  ## Default: appropriate/curl:latest
+  ##
+  image: appropriate/curl:latest
+
+  ## Deadline for job so it does not retry forever.
+  ## Default: activeDeadline: 300
+  ##
+  activeDeadline: 300
+
+  ## Specify the number of retries before considering job as failed.
+  ## https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#pod-backoff-failure-policy
+  ##
+  backoffLimit: 6
+
+  ## Hook delete policy for helm.
+  ## Default: hookDeletePolicy: hook-succeeded
+  ##
+  hookDeletePolicy: hook-succeeded
+
+  ## Restart policy for job
+  ## Default: OnFailure
+  restartPolicy: OnFailure
+
+  user:
+
+    ## The user name
+    ## Default: "admin"
+    username: "admin"
+
+    ## User password
+    ## single quotes must be escaped (\')
+    ## Default: (Randomly generated 10 characters of AlphaNum)
+    # password:
+
+    ## The user name and password are obtained from an existing secret. The expected
+    ## keys are `influxdb-user` and `influxdb-password`.
+    ## If set, the username and password values above are ignored.
+    # existingSecret: influxdb-auth
+
+    ## User privileges
+    ## Default: "WITH ALL PRIVILEGES"
+    privileges: "WITH ALL PRIVILEGES"
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources: {}
+#  requests:
+#    memory: 256Mi
+#    cpu: 0.1
+#  limits:
+#    memory: 16Gi
+#    cpu: 8
+resources:
+  limits:
+    cpu: 1000m
+    memory: 3096Mi
+  requests:
+    cpu: 1000m
+    memory: 3096Mi
+
+# Annotations to be added to InfluxDB pods
+podAnnotations: {}
+
+ingress:
+  enabled: false
+  tls: false
+  # secretName: my-tls-cert # only needed if tls above is true
+  hostname: influxdb.foobar.com
+  annotations: {}
+    # kubernetes.io/ingress.class: "nginx"
+    # kubernetes.io/tls-acme: "true"
+
+## Use an alternate scheduler, e.g. "stork".
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+# schedulerName:
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
+
+## Affinity for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+##
+affinity: {}
+
+## Tolerations for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: []
+# - key: "key"
+#   operator: "Equal|Exists"
+#   value: "value"
+#   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+## The InfluxDB image uses several environment variables to automatically
+## configure certain parts of the server.
+## Ref: https://hub.docker.com/_/influxdb/
+env:
```

Comment:
Automatically create `metrics` db. We have no authentication.

Reply:
Renamed to `testground` as this is what we use in the `sdk-go`.

Reply:
Ah, I wrote a comment about this above. Can we disable HTTP Auth as well?

Reply:
HTTP auth is disabled.